devops/makeLambdaFunction/makeLambdaFunction.yml

```diff
@@ -0,0 +1,26 @@
+Parameters: 
```

Comment:
I see that you've written your automation in `yml` while I wrote mine in `json`. We should decide on a consistent format moving forward for all of these CloudFormation templates. I'm happy to stick with `yml` unless other devops folks have a preference!

Reply:
Good point @dansjack! I personally feel like yml is easier to read because of less syntax, so it can be easier to see what is going on while configuring the template, but I am also happy to convert to json format in the future.

Reply:
Sounds good! Lets go with yml